collectors/test-results/jira-xray/src/main/java/com/capitalone/dashboard/core/client/testexecution/TestExecutionClientImpl.java

```diff
@@ -260,7 +261,7 @@ private void updateMongoInfo(List<Feature> currentPagedTestExecutions) {
      */
     private Map<String,Integer> getTestCountStatusMap(Feature testExec, Iterable<TestExecution.Test> tests) {
 
-        Map<String,Integer> map = new HashMap<String, Integer>(TEST_STATUS_COUNT_ATTRIBUTES.values().length);
+        Map<String,Integer> map = new HashMap<>(TEST_STATUS_COUNT_ATTRIBUTES.values().length);
```

Comment:
why are we limitting the size of hashmap? Alternative is to create the populated hashmap and make it immutable.

Reply:
no change needed here, reverted back to original.